canopen/pdo.py

```diff
@@ -398,11 +401,23 @@ def __init__(self, od):
         self.offset = None
         self.name = od.name
         self.length = len(od)
+        self._override_subindex = None
```

Comment:
Perhaps it would be better to just have `self.index` and `self.subindex` as normal attributes which are defaulted to `od.index` and `od.subindex` but are allowed to be overridden.

Reply:
After looking around I find that a bit inconsistent with how SDOs are handled.  Access to the index and subindex is always done through the `od` attribute of `common.Variable`.

For better encapsulation, we could wrap all those accesses into properties of `common.Variable` though, and just override the getter and setter in `pdo.Variable`.  That makes those `var.od.index` references a little easier to read and should be about equal performance-wise.

Reply:
Yes, it would be good to be able to access index and subindex directly in the common.Variable class. It seems overkill to use properties though.